css/style.css

```diff
@@ -1,34 +1,876 @@
+  line-height: 28px;
+  color: #ebebff;
+  text-align: center;
+  margin-bottom: 60px;
+}
+
+.form-element {
+  font-family: Poppins, sans-serif;
   font-style: normal;
+  font-weight: normal;
+  font-size: 17px;
+  line-height: 20px;
+  color: #172b4d;
+  flex: 1;
+  padding: 0 10px 0;
+  margin-bottom: 20px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  width: 80%;
+}
+
+input.form-element {
+  height: 48px;
+  width: 327px;
+}
+
+textarea.form-element {
+  height: 170px;
+  flex: 1 0 auto;
+}
+
+.form-element::placeholder {
+  color: #b3bac5;
+}
+
+.flex-auto {
+  flex: 1 0 auto;
+}
+
+.but:enabled {
+  background-color: #fff;
+  border: 1px solid #7f8cff;
+}
+
+.but:hover {
+  background-color: #fff;
+}
+
+.but:disabled {
+  color: #5e6c84;
+}
+
+.but:active {
+  background-color: #2230d2;
+}
+
+#but-ons {
+  width: 140px;
+  height: 48px;
+  border-radius: 8px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 450;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  letter-spacing: 0.03em;
+  color: #2230d2;
+}
+
+#but-ons:enabled {
+  background-color: #fff;
+  border: 1px solid #7f8cff;
+}
+
+#but-ons:hover {
+  background-color: #6070ff;
+}
+
+#but-ons:disabled {
+  color: #5e6c84;
+}
+
+#but-ons:active {
+  background-color: #2230d2;
+}
+
+/* Fix Mobile  */
+
+.grid-container {
+  /* display: -ms-grid; */
+  display: grid;
+
+  /* 3grid-template-columns: 2fr 2fr; */
+
+  padding: 30px 30px;
+
+  /* row-gap: repeat(nth-child(even), 5rem); */
+  margin-top: 5rem;
+  margin-bottom: 5rem;
+
+  /* border: solid; */
+
+  /* grid-template-columns: 50% 50%; */
+  grid-template-columns: 1fr;
+
+  /* width: 80%;
+  padding: 0%;
+  margin-left: 10%; */
+  grid-template-areas:
+    "item1"
+    "item2"
+    "item7"
+    "item4"
+    "item5"
+    "item6"
+    "item3"
+    "item8";
+}
+
+.grid-item1 {
+  grid-area: item1;
+
+  /* grid-row-start: span 2; */
+  border-bottom-right-radius: 0;
+  border-bottom-left-radius: 0;
+}
+
+.grid-item2 {
+  grid-area: item2;
+  margin-bottom: 5rem;
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+}
+
+.grid-item4 {
+  grid-area: item4;
+  margin-bottom: 5rem;
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+}
+
+.grid-item3 {
+  grid-area: item3;
+  border-bottom-right-radius: 0;
+  border-bottom-left-radius: 0;
+}
+
+.grid-item5 {
+  grid-area: item5;
+  border-bottom-right-radius: 0;
+  border-bottom-left-radius: 0;
 }
+
+.grid-item6 {
+  grid-area: item6;
+  margin-bottom: 5rem;
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+}
+
+.grid-item7 {
+  grid-area: item7;
+  border-bottom-right-radius: 0;
+  border-bottom-left-radius: 0;
+}
+
+.grid-item8 {
+  grid-area: item8;
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+}
+
+/* Add Desktop version Styling */
+
+/*
+
+@media (min-width: 768px) {
+  .menu {
+    display: none;
+  }
+
+  .portfolio,
+  .about,
+  .contact {
+    display: flex;
+  }
```

Comment:
- [ ]  Kindly consider removing this block of code as it doesn't make your codebase looks clean